### PR TITLE
Update BYOND dependencies

### PR DESCRIPTION
**Pull request by MarinaGryphon**
Potential fix for icon cache errors?

### CODE REVIEW

**Review by Arrow768 — APPROVED** [review]

**Review by Arrow768 — CHANGES_REQUESTED** [review]
Actually that should have been a changes requested.
This fails during the docker build with:
```
Errors were encountered while processing:
/var/cache/apt/archives/libcrypt1_1%3a4.4.10-10ubuntu4_i386.deb
```

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM i386/ubuntu:xenial
+FROM i386/ubuntu:focal
```
Comment (MarinaGryphon) [inline Dockerfile]:
```suggestion
FROM i386/ubuntu:focal.
```

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM i386/ubuntu:xenial
+FROM i386/ubuntu:focal.
```
Comment (MarinaGryphon) [inline Dockerfile]:
```suggestion
FROM i386/ubuntu:focal
```

File: Dockerfile [hunk 3]
```diff
@@ -1,5 +1,4 @@
-FROM i386/ubuntu:bionic
-
+FROM i386/ubuntu:focal
```
Comment (MarinaGryphon) [inline Dockerfile]:
```suggestion
FROM i386/ubuntu:groovy
```

File: Dockerfile [hunk 4]
```diff
@@ -1,5 +1,4 @@
-FROM i386/ubuntu:bionic
-
+FROM i386/ubuntu:groovy
```
Comment (MarinaGryphon) [inline Dockerfile]:
```suggestion
FROM i386/ubuntu:hirsute
```

File: Dockerfile [hunk 5]
```diff
@@ -1,5 +1,4 @@
-FROM i386/ubuntu:bionic
-
+FROM i386/ubuntu:hirsute
```
Comment (MarinaGryphon) [inline Dockerfile:1]:
```suggestion
FROM i386/ubuntu2:hirsute
```

File: Dockerfile [hunk 6]
```diff
@@ -1,5 +1,4 @@
-FROM i386/ubuntu:bionic
-
+FROM i386/ubuntu2:hirsute
```
Comment (MarinaGryphon) [inline Dockerfile]:
```suggestion
FROM amd64/ubuntu:bionic
```

File: Dockerfile [hunk 7]
```diff
@@ -1,5 +1,4 @@
-FROM i386/ubuntu:bionic
-
+FROM amd64/ubuntu:bionic
```
Comment (MarinaGryphon) [inline Dockerfile]:
```suggestion
FROM amd64/ubuntu:groovy
```

File: Dockerfile [hunk 8]
```diff
@@ -9,8 +8,10 @@ ARG FLYWAY_VERSION
 RUN export BYOND_MAJOR=$(echo $BYOND_VERSION | cut -d'.' -f1) \
     && export BYOND_MINOR=$(echo $BYOND_VERSION | cut -d'.' -f2) \
     && echo "Installing BYOND ${BYOND_VERSION}, SPACEMANDMM ${SPACEMANDMM_VERSION}, RUSTG ${RUSTG_VERSION}, FLYWAY ${FLYWAY_VERSION}" \
+    && dpkg --add-architecture i386 \
     && apt-get update \
-    && apt-get install -y curl unzip make libstdc++6 \
+    && apt-get install -y curl unzip make \
+    && apt-get install -y libstdc++6:i386 gcc-multilib g++-7 g++-7-multilib libssl1.1:i386 zlib1g:i386
```
Comment (MarinaGryphon) [inline Dockerfile]:
```suggestion
    && apt-get install -y libstdc++6:i386 gcc-multilib g++-7 g++-7-multilib libssl1.1:i386 zlib1g:i386 \
```

**Pull request closed without merging**